packages/cfd/src/sass/cfd.scss

```diff
@@ -174,6 +174,31 @@
             margin-bottom: 2.2rem;
         }
     }
+    &-info {
+        border-radius: 0.4rem;
+        background: $color-status-information;
+        display: flex;
+        padding: 0.8rem;
+        align-items: flex-start;
+        gap: 0.8rem;
+        margin-block-end: 0.8rem;
+    }
+    &__checkbox {
+        display: flex;
+        align-self: baseline;
+        margin-inline-start: -0.8rem;
+        margin-block-end: 3.2rem;
+        .dc-text {
+            font-size: var(--text-size-xxs);
+            text-align: var(--text-align-left);
+        }
```

Comment:
can't we handle these as props in `Text` component?

Reply:
I am reusing jurisdiction checkbox. All props are already set there. This way is easier.

Reply:
We can proceed with this for now but let's refactor it after. Seems like a lot of styles are being overridden.